nncf/onnx/graph/node_utils.py

```diff
@@ -172,9 +171,13 @@ def _get_activation_tensor_shape(
     :return: None, if there is no shape info, otherwise - tensor shape.
     """
     if target_point.type == TargetType.PRE_LAYER_OPERATION:
-        shape = nncf_graph.get_input_edges(node)[target_point.port_id].tensor_shape
+        edge = nncf_graph.get_input_edge_by_port_id(node, target_point.port_id)
```

Comment:
`if edge` condition was missed? Since `nncf_graph.get_input_edge_by_port_id` may return `None`.

Reply:
Well, maybe the introduction of the common method `nncf_graph.get_input_edge_by_port_id` was not a good suggestion from my side.
But, in the previous version, there was no input edge existing verification for the ONNX backend. Why do we need it now in new method?